### PR TITLE
Feature/output dir

### PR DESCRIPTION
**Pull request by cshcomcom**
## Description
- Add outputDirName parameter
- Write 'result.json' file

Now, we can set custom output directory name and see result.json file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ A benchmark framework to measure performance of [AI Network blockchain](https://
 - Usage
 ```
 npm install
-node ain_benchmark.js sample_config.json // Need to change the configuration
+node ain_benchmark.js <config_file> <output_directory[OPTIONAL]>
```
Comment (liayoo) [inline README.md:9]: Suggestion: how about node ain_benchmark.js <config_file> [<output_directory>] ?

File: ain_benchmark.js [hunk 2]
```diff
@@ -2,13 +2,14 @@ const ShardTest = require('./test_type/shard');
...
 const fs = require('fs');
+const moment = require('moment-timezone');
 
 process.on('uncaughtException', (err) => {
   console.log(`uncaughtException: ${JSON.stringify(err, null, 2)}`);
 });
 
 function checkArgs() {
-  if (process.argv.length !== 3) {
+  if (process.argv.length < 3 && process.argv.length > 4) {
```
Comment (liayoo) [inline ain_benchmark.js:12]: || ?

File: test_type/shard.js [hunk 3]
```diff
@@ -243,28 +241,26 @@ function printTestResult(testList) {
...
           `/ ${confirmJob.output.statistics.blockDuration / 1000} secs)`);
-      // console.log(`Loss Rate: ${confirmJob.output.statistics.lossRate}`);
-      // console.log(`Confirmed time average (ms) : ${confirmJob.output.statistics.confirmedTimeAverage}`); // TODO(csh): Delete after test
-      // console.log(`Confirmed time table (sec) : ${JSON.stringify(confirmJob.output.statistics.confirmedTimeTable, null, 2)}`); // TODO(csh): Delete after test
     }
     console.log();
   }
-  console.log(`Total TPS : ${totalTps.toFixed(5)}`);
+  totalTps = Number(totalTps.toFixed(5));
+  const lossRate = (totalTimeoutTxCount / totalTxCount * 100).toFixed(5) + '%';
```
Comment (liayoo) [inline test_type/shard.js:251]: How about keeping lossRate as a number like totalTps? You could add the '%' in console.log

File: README.md (hunk 1 above)
Comment (cshcomcom) [reply README.md:9]: Looks good! I'll change it

File: ain_benchmark.js (hunk 2 above)
Comment (cshcomcom) [reply ain_benchmark.js:12]: Thanks!

File: test_type/shard.js (hunk 3 above)
Comment (cshcomcom) [reply test_type/shard.js:251]: Looks better! I'll change it

**Comment by cshcomcom** [comment]
All comments are addressed! PTAL @liayoo

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

**Comment by cshcomcom** [comment]
Thanks for the review! Now merging

**Pull request merged**